From c7bd3504e6a791e136d909ab818d77bcecd6b982 Mon Sep 17 00:00:00 2001
From: Adarsh Liju Abraham <adarsh.liju.abraham@gmail.com>
Date: Mon, 6 Jun 2022 03:06:02 +0530
Subject: [PATCH] Change README

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 4e31d2f..ad7e80b 100644
--- a/README.md
+++ b/README.md
@@ -1 +1 @@
-# sample
+# sample repo for testing
-- 
2.34.1

